### DISCUSSION TITLE
Show publisher details

### DISCUSSION THREAD

**Discussion by jackuars7** [tnodir/fort#167] [Ideas]
Is it possible to show the publisher details in the main GUI, so that I can grant access to most Microsoft related services at one go. It makes organizing firewall rules much easier.

**Comment by tnodir** [comment]
Do you mean a "Programs" window?

You can select a program and see publisher details at bottom status panel:

![fort-status](https://github.com/tnodir/fort/assets/303655/323cc2ab-ab7b-4665-bd15-a63498d95f75)
